modular_citadel/code/modules/reagents/chemistry/reagents/fermi_reagents.dm

```diff
@@ -0,0 +1,1596 @@
+			var/mob/living/carbon/C = get_mob_by_key(creatorID)
+			if(M.has_status_effect(STATUS_EFFECT_INLOVE))
+				return
+			if((C in viewers(7, get_turf(M))) && (C.client))
+				M.reagents.remove_reagent(src.id, src.volume)
+				FallInLove(C, M)
+			return
+		if(volume < 1)//You don't get to escape that easily
+			FallInLove(pick(GLOB.player_list), M)
+	if (M.ckey == creatorID && creatorName == M.real_name)//If you yourself drink it, it does nothing.
+		return
+	if(!M.client)
+		metabolization_rate = 0 //Stops powergamers from quitting to avoid affects. but prevents affects on players that don't exist for performance.
+		return
+	if(metabolization_rate == 0)
+		metabolization_rate = 0.5
+	var/datum/status_effect/chem/enthrall/E = M.has_status_effect(/datum/status_effect/chem/enthrall)//If purity is over 5, works as intended
+	if(!E)
+		return
+	else
+		E.enthrallTally += 1
+	if(prob(1))
+		M.adjustBrainLoss(1)//Honestly this could be removed, in testing it made everyone brain damaged, but on the other hand, we were chugging tons of it.
+	..()
+
+/datum/reagent/fermi/enthrall/overdose_start(mob/living/carbon/M)//I made it so the creator is set to gain the status for someone random.
+	. = ..()
+	if (M.ckey == creatorID && creatorName == M.real_name)//If the creator drinks 100u, then you get the status for someone random (They don't have the vocal chords though, so it's limited.)
+		to_chat(M, "<span class='love'><i>You are unable to resist your own charms anymore, and become a full blown narcissist.</i></span>")
+		/*Old way of handling, left in as an option B
+		var/list/seen = viewers(7, get_turf(M))//Sound and sight checkers
+		for(var/mob/living/carbon/victim in seen)
+			if(victim == M)//as much as I want you to fall for beepsky, he doesn't have a ckey
+				seen = seen - victim
+			if(!victim.ckey)
+				seen = seen - victim
+		var/mob/living/carbon/chosen = pick(seen)
+		creatorID = chosen.ckey
+		if (chosen.gender == "female")
+			creatorGender = "Mistress"
+		else
+			creatorGender = "Master"
+		creatorName = chosen.real_name
+		creator = get_mob_by_key(creatorID)
+		*/
+	M.add_trait(TRAIT_PACIFISM, "MKUltra")
+	var/datum/status_effect/chem/enthrall/E
+	if (!M.has_status_effect(/datum/status_effect/chem/enthrall))
+		M.apply_status_effect(/datum/status_effect/chem/enthrall)
+		E = M.has_status_effect(/datum/status_effect/chem/enthrall)
+		E.enthrallID = creatorID
+		E.enthrallGender = creatorGender
+		E.master = creator
+	else
+		E = M.has_status_effect(/datum/status_effect/chem/enthrall)
+	to_chat(M, "<span class='big love'><i>Your mind shatters under the volume of the mild altering chem inside of you, breaking all will and thought completely. Instead the only force driving you now is the instinctual desire to obey and follow [creatorName]. Your highest priority is now to stay by their side at all costs.</i></span>")
+	M.slurring = 100
+	M.confused = 100
+	E.phase = 4
+	E.mental_capacity = 0

```

Comment:
even as an adminspawn reagent why is this a thing?

Reply:
Because I thought it'd be cute to have cats.

Reply:
Also I changed it so that you only get cat ears now, no race/gender change.